pom.xml

```diff
@@ -61,6 +69,10 @@
                         </goals>
                     </execution>
                     <execution>
+                        <id>compile</id>
+                        <goals> <goal>compile</goal> </goals>
+                    </execution>
+                    <execution>
```

Comment:
The four lines here duplicate what you already have aboce them and leads to a failing build. You have to remove them.

:bulb: Always run the build locally to make sure everything works as expected.